packages/web/src/components/nav/desktop/LeftNavDroppable.module.css

```diff
@@ -0,0 +1,37 @@
+.droppableLink {
```

Comment:
not super elegant, but you could use this for LeftNavDroppable: https://emotion.sh/docs/class-names

Reply:
I actually did this in another place and I can try to replicate it here. `Droppable` should probably be refactored into a render function pattern, but that's too big for this PR.